### DISCUSSION TITLE
Enum_List and Enumeration_List indentation?

### DISCUSSION THREAD

**Discussion by stureandersen** [NilsSve/DfRefactor#47] [Ideas]
Enum_List/End_Enum_List and Enumeration_List/End_Enumeration_List should indent the same (they are identical)

**Comment by NilsSve** [comment]
This is odd and looks like a bug.
As both Enum_List & Enumeration_List are part of the "Scope Start - Scop End" list, they should behave in the same manner.

For some reason the Enumeration_List members are all "indented" to column 0, while the Enum_List members are indented correctly.

**Comment by NilsSve** [comment]
I'm closing this as I have instead opened it as an "Issue" with the same header.
